integration/generic-tests/kexec_test.go

```diff
@@ -33,6 +33,38 @@ func TestMountKexec(t *testing.T) {
 		},
 		QEMUOpts: qemu.Options{
 			Timeout: 20 * time.Second,
+			Devices: []qemu.Device{
+				qemu.ArbitraryArgs{"-m", "8192"},
+			},
+		},
+	})
+	defer cleanup()
+
+	if err := q.Expect("SAW KEXEC=Y"); err != nil {
+		t.Fatal(err)
+	}
+}
+
+// TestMountKexecLoad is same as TestMountKexec except it test calling
+// kexec_load syscall than file load.
+func TestMountKexecLoad(t *testing.T) {
+	// TODO: support arm
+	if vmtest.TestArch() != "amd64" && vmtest.TestArch() != "arm64" {
```

Comment:
It works on arm64? I thought you implemented bzimage only, not ELF

Reply:
Done.

no. removed  `arm64`. This was from my early sketch.